### PR TITLE
Add question error panel

### PR DESCRIPTION
**Pull request by LJBabbage**
### What is the context of this PR?
Add an error panel in the template for question errors. 

### How to review 
Check that a question is wrapped in a red border panel on error. This will affect both mutually exclusive and date range 

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Review by berroar — APPROVED** [review]
Error panel is now being displayed correctly

File: templates/partials/question.html [hunk 1]
```diff
@@ -107,5 +110,23 @@
   {{ individual_response_guidance }}
   {{ question_definition }}
   {{ question_guidance }}
-  {{ question_answers }}
+
+  {% if question_error %}
+    {% set config = {
+      "text": question_error,
+      "id": question.id ~ '-error',
```
Comment (MebinAbraham) [inline templates/partials/question.html:117]: The question error anchor link needs to be updated to anchor to the error panel. It currently still links to the question itself. Checklist item 3 on the card for more info.

We already have a method for it which is just used fo answers at the moment.
https://github.com/ONSdigital/eq-questionnaire-runner/blob/master/app/forms/questionnaire_form.py#L386-L387

**Review by MebinAbraham — CHANGES_REQUESTED** [review]
Anchor link needs fixing but apart from that, the error panel seems to be as expected.

File: templates/partials/question.html (hunk 1 above)
Comment (LJBabbage) [reply templates/partials/question.html:117]: I saw it linked to the question and thought that was fine, however I didn't appreciate that the answer links to the error panel and not the answer, so will bring it inline.

**Review by MebinAbraham — APPROVED** [review]

**Pull request merged**